ChangeLog

```diff
@@ -1,4 +1,49 @@
-* 2016-??-??: Version 1.0.0
+* 2016-??-??: Version 1.0.0b2
+  * Collect coverage information from the unit tests
```

Comment:
It might be nicer in future to have 2 sections to the changelog: Updates and Bugfixes?